Clarify case requirement for sign-on URL

martinbherbener opened the pull request:
Per Parakh Jain <parja@microsoft.com>, reply URL (therefore effectively sign-on URL) must contain tenant name as lower-case in order to avoid error AADSTS50011: Reply address '' specified by the request is not a valid URL. Allowed schemes: 'http,https'

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

martinbherbener committed:
Clarify case requirement for sign-on URL

Per Parakh Jain <parja@microsoft.com>, reply URL (therefore effectively sign-on URL) must contain tenant name as lower-case in order to avoid error AADSTS50011: Reply address '' specified by the request is not a valid URL. Allowed schemes: 'http,https'

parakhj commented on articles/active-directory-b2c/active-directory-b2c-setup-aad-custom.md:
```diff
@@ -50,6 +50,9 @@ To enable sign-in for users from a specific Azure AD organization, you need to r
 1. For **Sign-on URL**, enter the following URL, where `yourtenant` is replaced by the name of your Azure AD B2C tenant (`fabrikamb2c.onmicrosoft.com`):
 
+>[!NOTE]
+> The value for "yourtenant" must be all lowercase in the Sign-on URL regardless of the case used within B2C.
```
@martinbherbener Thanks for adding this! Could you update it to say "The value for "yourtenant" must be all lowercase in the Sign-on URL" (just remove the ending)
And also indent it one right?

martinbherbener committed:
Update active-directory-b2c-setup-aad-custom.md

martinbherbener reviewed (commented):
This ok?  Not sure I indented as you wanted.

GitHubber17 committed:
Update active-directory-b2c-setup-aad-custom.md
